tests/wpt/css-tests/css-transforms-1_dev/html/reference/transform3d-sorting-006-ref.htm

```diff
@@ -11,20 +12,18 @@
     -->
   </head>
   <body>
-    <div style="height: 70px; width: 50px; background: red;
-      border-bottom: 30px solid black;
-      transform: rotateX(45deg) rotateY(45deg);
-      transform-origin: right"></div>
     <div style="height: 70px; width: 50px; background: green;
-      border-bottom: 30px solid black;
-      transform: translate(50px, -100px) rotateX(45deg) rotateY(-45deg);
-      transform-origin: left"></div>
+      border-top: 30px solid black;
+      transform: rotateX(45deg) rotateY(-45deg);
+      transform-origin: right"></div>
     <div style="height: 100px; width: 50px; background: red;
+      transform: translateY(-100px) rotateX(45deg) rotateY(45deg);
+      transform-origin: right"></div>
```

Comment:
Actually, given these tests are in wpt, I don't think we should change them at all. I guess these pass on both Blink and Gecko?